example/lib/main.dart

```diff
@@ -36,7 +36,7 @@ class _MyAppState extends State<MyApp> {
 
   final AuthConfig _config = AuthConfig(
       clientId: "014ae4bd048734ca2dea",
-      endpoint: "door.casdoor.com",
+      endpoint: "door.casdoor.com",// use endpoint: "http://door.casdoor.com" to support http scheme
```

Comment:
Refer to: https://github.com/casdoor/casdoor-js-sdk#:~:text=%3A%20SdkConfig%20%3D%20%7B-,serverUrl,-%3A%20%22https%3A//door.casbin

Rename to `serverUrl`, and add procotol by default.

So finally:

```
serverUrl: "https://door.casdoor.com"
```